gradle/libs.versions.toml

```
@@ -1,40 +1,40 @@
 [versions]
-activityCompose = "1.9.3"
+activityCompose = "1.10.1"
 androidGradlePlugin = "8.7.3"
 androidXBrowser = "1.8.0"
 coilBOM = "2.5.0"
-composeBOM = "2024.11.00"
+composeBOM = "2025.04.00"
 androidxCamera = "1.4.2"
 androidxComposeCompiler = "1.5.12"
-androidxCore = "1.15.0"
+androidxCore = "1.16.0"
 androidxEspresso = "3.6.1"
 androidxHiltNavigationCompose = "1.2.0"
 androidxLifecycle = "2.8.7"
-androidxMaterialIcons = "1.7.5"
+androidxMaterialIcons = "1.7.8"
 androidxTestExt = "1.2.1"
 androidXTestRunner = "1.6.2"
 androidXTestRules = "1.6.1"
 androidxWindow = "1.3.0"
 binaryCompatibilityValidator = "0.17.0"
 compileSdk = "35"
-compose-navigation = "2.8.4"
+compose-navigation = "2.8.9"
 commonMark = "0.22.0"
 dokka = "2.0.0"
 hilt = "2.55"
 hiltExt = "1.2.0"
 junit = "4.13.2"
-kotlin = "2.1.0"
-ksp = "2.1.0-1.0.29"
-media3Exoplayer = "1.5.0"
-minSdk = "26"
+kotlin = "2.1.20"
+ksp = "2.1.20-1.0.32"
```

New release added [last week](https://github.com/google/ksp/releases) :) 

```suggestion
ksp = "2.1.20-2.0.0"
```

---

@shubham7109 as per our discussion we are not upgrading this version now as we are seeing build failure with `FeatureFormApp`

https://github.com/google/ksp/issues/2407
https://github.com/google/ksp/pull/2414/files